CMake/resolve_dependency_modules/simdjson.cmake

```
@@ -27,6 +27,16 @@ message(STATUS "Building simdjson from source")
 FetchContent_Declare(
   simdjson
   URL ${VELOX_SIMDJSON_SOURCE_URL}
-  URL_HASH ${VELOX_SIMDJSON_BUILD_SHA256_CHECKSUM})
+  URL_HASH ${VELOX_SIMDJSON_BUILD_SHA256_CHECKSUM}
+  SOURCE_DIR ${CMAKE_BINARY_DIR}/_deps/simdjson-src/simdjson)
```

There is an existing option for what you are doing here that is not brittle against changes to the soruce path (that can happen through user level changes)
```suggestion
  SOURCE_SUBDIR simdjson)
```